redislock.go

```diff
@@ -160,13 +187,16 @@ func (l *Lock) Metadata() string {
 
 // TTL returns the remaining time-to-live. Returns 0 if the lock has expired.
```

Comment:
we should mention that it returns the minimum TTL if multiple keys are held by the lock

Reply:
Done !